tests/features/subscriptions.feature

```diff
@@ -0,0 +1,61 @@
+@api
+Feature: Subscription
+  In order to be notified on content created or updated on the website
+  As an authenticated user
+  I want to be able to subscribe to content and manage subscriptions.
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+    And the module is enabled
+      |modules                      |
+      |multisite_notifications_core |
+  @javascript
```

Comment:
Maybe put a blank line before @javascript, to make it visually more separate from the Background